src/Tools/Source/RunTests/Options.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace RunTests
+{
+    internal class Options
+    {
+        public bool UseHtml { get; set; }
+
+        public bool Test64 { get; set; }
+
+        public string Trait { get; set; }
+
+        public string NoTrait { get; set; }
+
+        public string[] Assemblies { get; set; }
+
+        public string XunitPath { get; set; }
+
+        internal static Options Parse(string[] args)
+        {
+            if (args == null || args.Any(a => a == null) || args.Length < 2)
+            {                
+                return null;
+            }
+
+            var opt = new Options {XunitPath = args[0], UseHtml = true};
+
+            int index = 1;
+                    
+            var comp = StringComparer.OrdinalIgnoreCase;
+            while (index < args.Length)
+            {
+                var current = args[index];
+                if (comp.Equals(current, "-test64"))
+                {
+                    opt.Test64 = true;
+                    index++;
+                }
+                else if (comp.Equals(current, "-xml"))
+                {
+                    opt.UseHtml = false;
+                    index++;
+                }
+                else if (current.Length > 7 && current.StartsWith("-trait:", StringComparison.OrdinalIgnoreCase))
```

Comment:
comp is a StringCompar**er**, StartsWith expects a StringCompar**ison**. I don't see how I could reuse comp with StartsWith.